x-pack/plugins/security_solution/public/overview/components/detection_response/soc_trends/translations.ts

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const SOC_TRENDS = i18n.translate('xpack.securitySolution.detectionResponse.socTrends', {
+  defaultMessage: 'SOC Trends',
+});
+
+export const CASES_MTTR_STAT = i18n.translate('xpack.securitySolution.detectionResponse.mttr', {
+  defaultMessage: 'Avg. case response time',
```

Comment:
Does shorthand ('Avg') get translated? Would it be easier to make it the full text "Average"?

Reply:
going off of design, will make the update and see if they notice :wink: